extension/src/prototypes/ui-components/HelperText.tsx

```diff
@@ -0,0 +1,25 @@
+import { useFormControlContext } from "@mui/base/FormControl";
```

Comment:
Sorry, I missed to tell you about this, but I want to keep `prototypes` directory be only ported files from the Takram's prototype.
So could you move `HelperText` and `InputLabel` to `/shared/ui-components`?